blocks/cards/cards.css

```diff
@@ -198,6 +198,10 @@ main .cards.icons ul::-webkit-scrollbar {
   display: none;
 }
 
+.cards.icons li {
+  width: 25%;
```

Comment:
This will likely break for any number of cards other than 4. I don't know if there is a way to enforce equal size otherwise.

Reply:
yes, true - I was at the same point and couldn't find a better way for equal size - but I'll check once more

Reply:
@trieloff Should work now with any number of icons